src/pages/page/[page].test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render, screen } from '@testing-library/react';
+
+import { postListTestIds } from 'components/Post/List';
+import { BASIC_FIELDS, getAllPosts } from 'lib/post';
+import Page, { pageDataTestIds } from 'pages/page/[page].page';
+
+describe('Page', () => {
+  it('renders the heading', () => {
+    const posts = getAllPosts(BASIC_FIELDS);
+
+    render(<Page posts={posts} currentPage={1} totalPages={1} />);
+
+    const heading = screen.getByTestId(pageDataTestIds.heading);
+
+    expect(heading).toBeVisible();
+  });
+
+  describe('PostList', () => {
+    describe('when there are no posts', () => {
+      it('does not render the list', () => {
```

Comment:
```suggestion
      it('does NOT render the list', () => {
```

@hoangmirs Should we emphasis this kind of word like we did in other language test suite?